docker-for-mac/release-notes.md

```diff
@@ -13,10 +13,28 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.0.4
+2020-07-27
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
```

Comment:
@stephen-turner How about these URLs? Do we have an equivalent desktop download URL?

Reply:
No, because this is pointing to a Hub page not the actual binary